src/Microsoft.DotNet.CodeAnalysis/PackageFiles/PinvokeAnalyzer.props

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup Condition="'$(OSGroup)'=='Windows_NT'">
+    <!-- This file helps us choose the correct allowed pinvoke file.
+       1. In case we are building for windows and TargetGroup is Netcore50 or netcore50aot (Store profile) we use OneCoreUWPApis.txt
+       2. For all other windows configuration we use OneCoreAPIs.txt file.
+    -->
+    <ValidPinvokeMappings Condition="'$(TargetGroup)'==''">$(MSBuildThisFileDirectory)PinvokeAnalyzer_OneCoreApis.txt</ValidPinvokeMappings>
```

Comment:
Given the ties in with other common properties like CodeAnalysisRuleSet I would merge this into common code analyzer targets file. Plus there isn't enough setting in here to keep them separate in my opinion.